trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/compute_broadband_noise.py

```diff
@@ -0,0 +1,111 @@
+## @ingroupMethods-Noise-Fidelity_One-Propeller
+# compute_broadband_noise.py
+#
+# Created:  Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units , Data 
+import numpy as np 
+ 
+from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.dbA_noise   import A_weighting  
+from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools             import SPL_harmonic_to_third_octave
+
+# ----------------------------------------------------------------------
+# Frequency Domain Broadband Noise Computation
+# ----------------------------------------------------------------------
+## @ingroupMethods-Noise-Fidelity_One-Propeller
+def compute_broadband_noise(freestream,angle_of_attack,position_vector,
+                            velocity_vector,propeller,auc_opts,settings,res):
+    '''This computes the broadband noise of a propeller or rotor in the frequency domain
+    
+    Assumptions:
+        Coorelated are adjusted from reference altitude of 300 ft 
+
+    Source:
+       Schlegel, Ronald, Robert King, and Harold Mull. Helicopter rotor noise generation 
+       and propagation. UNITED TECHNOLOGIES CORP STRATFORD CT SIKORSKY AIRCRAFT DIV, 1966.
+    
+    
+    Inputs:  
+        freestream                    - freestream data structure
+        angle_of_attack               - aircraft angle of attack
+        position_vector               - position vector of aircraft
```

Comment:
maybe specify the shape? Its 4D but specify what each dimension corresponds to. I think it's [number control points/aoa,   number of mics,   number of props,   positions]?

Reply:
Thanks, I added it.